apps/src/code-studio/pd/workshop_enrollment/workshop_details.jsx

```diff
@@ -82,6 +82,21 @@ export default class WorkshopDetails extends React.Component {
     );
   }
 
+  fee() {
+    if (this.props.workshop.course === 'CS Fundamentals') {
+      return (
+        <div className="row">
+          <div className="span2" style={styles.label}>
+            <strong>Fee:</strong>
+          </div>
+          <div className="span2">{this.props.workshop.fee || 'No cost!'}</div>
+        </div>
+      );
+    }
+
+    return null;
+  }
+
```

Comment:
It's also painful that we have to hand-craft new React code for new fields in a form, when really we should just have to add a single entry (say in a server-side JSON file) to drive its appearance in the user-facing form.